src/App.js

```diff
@@ -194,6 +214,7 @@ class App extends Component {
     for (let i = 0; i < list.length; i++) {
       this.requestDeviceStatus(list[i]);
     }
```

Comment:
still can't get rid of that guy? :) I don't know if you read my comment in the other commit. Maybe I was wrong?